cli/flags.go

```diff
@@ -734,6 +734,48 @@ func (f *FlagSection) Uint64Var(i *Uint64Var) {
 	})
 }
 
+// LoggerConfigurer is the interface to configure a logger.
+type LoggerConfigurer interface {
+	SetLevel(level string)
+	SetMode(mode string)
+}
+
+// LoggerVar binds common logging flags to a [LoggerConfigurer].
+func (f *FlagSection) LoggerVar(i LoggerConfigurer) {
+	parser := func(s string) (string, error) { return s, nil }
+	printer := func(v string) string { return v }
+	var dummy string
+
+	Flag(f, &Var[string]{
+		Name:    "log-level",
+		Usage:   "To specify the log level with one of debug|info|warn|error|fatal",
```

Comment:
Nit: it might be better to do something like:

```go
levels := []string{"debug", "info", "warn", "error", "fatal"}

//
Usage: "To specify the log level with one of " + strings.Join(levels, "|")
//
Predict: predict.Set(levels)
```